editor/plugins/polygon_2d_editor_plugin.cpp

```diff
@@ -52,38 +53,37 @@
 #include "scene/gui/texture_rect.h"
 #include "scene/gui/view_panner.h"
 
-class UVEditDialog : public AcceptDialog {
-	GDCLASS(UVEditDialog, AcceptDialog);
-
-	void shortcut_input(const Ref<InputEvent> &p_event) override {
-		const Ref<InputEventKey> k = p_event;
-		if (k.is_valid() && k->is_pressed()) {
-			bool handled = false;
-
-			if (ED_IS_SHORTCUT("ui_undo", p_event)) {
-				EditorNode::get_singleton()->undo();
-				handled = true;
-			}
-
-			if (ED_IS_SHORTCUT("ui_redo", p_event)) {
-				EditorNode::get_singleton()->redo();
-				handled = true;
-			}
-
-			if (handled) {
-				set_input_as_handled();
-			}
-		}
-	}
-};
-
 Node2D *Polygon2DEditor::_get_node() const {
```

Comment:
This method looks like it can be removed.

Reply:
It's override from `AbstractPolygon2DEditor`